haskell/1/index.md

```diff
@@ -1,23 +1,14 @@
 My name is Chris. I teach Haskell to people that are new to programming and as well as long-time coders. Haskell is a general purpose programming language that is most useful to mere mortals.
 
-I'm going to show you how to write a package in Haskell and interact
-with the code inside of it.
+I'm going to show you how to write a package in Haskell and interact with the code inside of it.
 
 
 ## Installing tools for writing Haskell code
 
 
-The most popular compiler for Haskell is `GHC` and you use `Cabal` alongside `GHC` to manage projects and their dependencies. Packaging itself is part of `GHC` via `ghc-pkg`.
+We're going to use Stack to manage our project dependencies, compiler, building our code, and running our tests. Start by [getting Stack installed](http://haskellstack.org).
```

Comment:
Perhaps link directly to the installation page? http://docs.haskellstack.org/en/stable/install_and_upgrade/